### PR TITLE
Bufr tables latest

### PR DESCRIPTION
**Pull request by david-i-berry**
Docker file updated to add Debian bookworm to source list and to then install the latest version of the ecCodes BUFR tables from bookworm. The tables are not available as part of bullseye.

### CODE REVIEW

File: wis2box-ctl.py [hunk 1]
```diff
@@ -184,6 +184,7 @@ def make(args) -> None:
         _ = run(args, split('docker ps -a -q'), asciiPipe=True)
         run(args, split(f'docker rm {_}'))
     elif args.command == "restart":
+        print(f"Restarting {containers}")
```
Comment (tomkralidis) [inline wis2box-ctl.py:187]: Remove print statement.

**Review by tomkralidis — CHANGES_REQUESTED** [review]
Thanks @david-i-berry.  Additional questions:
- any idea on what the release lifecycle of libeccodes-data is on bullseye?
- is there any impact to the resulting image size once built?

File: wis2box-ctl.py (hunk 1 above)
Comment (david-i-berry) [reply wis2box-ctl.py]: That shouldn't be there, I appear to have contaminated my main branch. The latest commit should undo that.

**Comment by david-i-berry** [comment]
re lifecycle of libeccodes-data on bullseye. I don't know, I have no experience or knowledge of how the packages / sources are managed within Debian etc.
 
Image size will only be a bit larger due to the addition of the latest BUFR tables but there is no way to avoid this.

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**